pieces.py

```diff
@@ -139,7 +182,9 @@ def serialize(self, element):
         if self.author is not None:
             etree.SubElement(m, 'author').text = self.author
         if self.description is not None:
-            etree.SubElement(m, 'description').text = self.description
+            desc = etree.SubElement(m, 'description')
```

Comment:
What do you mean with uniform in this case? Can you give an example of how the XML should look like?